dial_opts_test.go

```diff
@@ -2,6 +2,7 @@ package grpcutil_test
 
 import (
 	"log"
+	"google.golang.org/grpc/credentials/insecure"
```

Comment:
nit: modules from third-party providers should all be grouped

(`gofumpt` rules)